### PR TITLE
XP-248 use structlog

### PR DESCRIPTION
**Pull request by tanertopal**
### References
Resolves XP-248

### Summary
PR make sure all logging happens with structlog logger 

### Are there any open tasks/blockers for the ticket?
None

### Next Step
None

### Reviewer checklist

*Reviewer agreement:*
* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [ ] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

File: xain_fl/fl/participant/participant.py [hunk 1]
```diff
@@ -72,7 +71,7 @@ def train_round(
                 and optimizer configs
         """
         logger.info(
-            "Participant %s: train_round START (epoch_base=%s)", self.cid, epoch_base
+            f"Participant {self.cid}: train_round START (epoch_base={epoch_base})"
```
Comment (atymoshchuk) [inline xain_fl/fl/participant/participant.py]: Please remove f-strings

File: benchmarks/ops/run.py [hunk 2]
```diff
@@ -135,4 +139,4 @@ def ec2(image: str, timeout: int = 300, instance_cores=2, **kwargs):
         AdditionalInfo=absl_flags,  # Helpful to identify instance in EC2 UI
     )
     instance_id = run_response["Instances"][0]["InstanceId"]
-    logging.info({"InstanceId": instance_id, "Name": instance_name})
+    logger.info({"InstanceId": instance_id, "Name": instance_name})
```
Comment (janpetschexain) [inline benchmarks/ops/run.py]: missing message string as first argument

File: xain_fl/fl/participant/participant.py (hunk 1 above)
Comment (janpetschexain) [inline xain_fl/fl/participant/participant.py]: separate logging message and arguments

Comment (finiteprods) [reply xain_fl/fl/participant/participant.py]: remind us what's wrong with f-strings again?

**Pull request merged**